WeakAuras/WeakAuras.lua

```diff
@@ -3862,12 +3864,30 @@ local wrappedGetOverlayInfo = wrapTriggerSystemFunction("GetOverlayInfo", "table
 
 Private.GetAdditionalProperties = function(data)
   local props = {}
-  for i = 1, #data.triggers do
-    local triggerSystem = GetTriggerSystem(data, i);
-    if (triggerSystem) then
-      local triggerProps = triggerSystem.GetAdditionalProperties(data, i)
-      if triggerProps then
-        props[i] = triggerProps
+  local dataCollection = {data}
```

Comment:
you should use TraverseLeafsOrAura, which does give you all leafs nodes (if its a group), or the aura itself if it isn't a group.

Reply:
Thanks! I should have known that existed an hour ago.